bin/pyxtrlock

```diff
@@ -23,6 +24,8 @@ try:
 except ImportError as err:
     panic(err)
 
+detect_wayland_session()
```

Comment:
I’d change the naming to make it clear that it enforces things. For example: ``require_x11_session``. Otherwise, I’m wondering "you’re detecting wayland, and then what? manipulate some non-obvious global state?"

Reply:
Makes sense, thanks.